client/js/components/content-slider.js

```diff
@@ -365,6 +364,14 @@ const contentSlider = (el, options) => {
   sliderElements.prevControl.addEventListener('click', prevSlide, true);
   sliderElements.nextControl.addEventListener('click', nextSlide, true);
 
+  nodeList(sliderElements.slideItems).forEach((item) => {
+    item.addEventListener('click', ({ target, currentTarget }) => {
+      if (!target.matches('img')) return;
```

Comment:
Will add a note (because we want to get the data-attribute from the slide item).